rkt/prepare.go

```diff
@@ -61,13 +64,12 @@ func runPrepare(args []string) (exit int) {
 		}
 	}
 
-	appArgs, images, err := parseAppArgs(args)
-	if err != nil {
-		stderr("prepare: error parsing app image arguments")
+	if err = parseApps(&rktApps, args, &prepareFlags); err != nil {
```

Comment:
tests and the ability to potentially seed global stuff in *apps.Apps before calling parseApps()